build_tools/waf_dynamo.py

```diff
@@ -1540,7 +1548,7 @@ def detect(conf):
             conf.env['LUA_BYTECODE_ENABLE_32'] = 'yes'
 
     conf.env['STATICLIB_CARES'] = []
-    if platform != 'web':
+    if platform not in ['js-web', 'wasm-web']:
```

Comment:
Previously, the compiler happily ignored the fact that the library didn't exist for that platform.